lib/index.js

```diff
@@ -50,6 +50,13 @@ const allLevels = [
   '!/licence{,.*[^~$]}',
 ]
 
+const rootOnly = [
+  'readme',
+  'license',
+  'licence',
+  'copying',
+]
```

Comment:
does it maybe make more sense for these to be regexes?

Reply:
Yes, I can look into that. I'll likely have to make changes to the excludeNonRoot function to accommodate for the change.

Reply:
This is fine, let's keep an explicit list which is much easier to reason about at a glance.